scripts/install-server.sh

```
@@ -486,7 +486,132 @@ ensure_container_runtime() {
+Description=taOS: keep incus bridge forwarding open alongside Docker
+After=docker.service incus.service
+Wants=docker.service
+[Service]
+Type=oneshot
+ExecStart=/bin/sh -c 'iptables -C FORWARD -i incusbr0 -j ACCEPT 2>/dev/null || iptables -I FORWARD -i incusbr0 -j ACCEPT; iptables -C FORWARD -o incusbr0 -j ACCEPT 2>/dev/null || iptables -I FORWARD -o incusbr0 -j ACCEPT'
+[Install]
+WantedBy=multi-user.target
+UNIT
+        sudo systemctl daemon-reload >/dev/null 2>&1 || true
+        sudo systemctl enable taos-incus-docker-net.service >/dev/null 2>&1 \
+            && log "installed taos-incus-docker-net.service (persists incusbr0 rules across reboots)" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)